### PR TITLE
Technology

### PR DESCRIPTION
**Pull request by DavidRojas1612**
## Summary
- Created tech css file, for styles of the techs component.
- Layout of the card technology 
## Screenshots

image

## Trello Card
https://trello.com/c/JEZZkfvN

### CODE REVIEW

File: maquetado/scss/styles.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .section-project {
   padding: 2rem;
 }
+
+.m-bottom-1 {
+  margin: 0 0 1rem 0;
+}
```
Comment (alejobh) [inline maquetado/scss/styles.scss]: Add EOF

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    border-radius: 5px;
+    content: '';
+    height: 100%;
+    left: -10px;
+    position: absolute;
+    top: -10px;
+    width: 100%;
+  }
+
+  &-logo {
```
Comment (alejobh) [inline maquetado/scss/tech.scss]: I really liked this ampersand touching! great work 👍

File: maquetado/index.html [hunk 3]
```diff
@@ -10,7 +10,17 @@
...
+            <div class="tech-box">
+                  <article class="tech">
+                      <img src="../assets/react_icon.svg" alt="" class="tech-logo">
+                      <div class="tech-info">
+                          <h3 class="m-bottom-1">ReactJS</h3>
+                          <span class="tech-release m-bottom-1">Release: 2013</span>
+                          <p class="tech-description">React is a declarative, component-based JavaScript libar for building user interfaces</p>
+                      </div>
+                  </article>
+            </div>
```
Comment (CarlosHWolox) [inline maquetado/index.html]: Take care with indentation.

File: maquetado/index.html [hunk 4]
```diff
@@ -10,7 +10,17 @@
   <body>
       <main class="main-project">
           <section class="section-project">
-            <h1 class="main-title">Front-End Tools</h1>
+            <h1 class="main-title m-bottom-1">Front-End Tools</h1>
+            <div class="tech-box">
+                  <article class="tech">
+                      <img src="../assets/react_icon.svg" alt="" class="tech-logo">
```
Comment (CarlosHWolox) [inline maquetado/index.html]: add an alt.

File: maquetado/scss/tech.scss (hunk 2 above)
Comment (CarlosHWolox) [reply maquetado/scss/tech.scss]: Wow! I didn't know this, good work!

File: maquetado/scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
+%equal-align {
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: cool

File: maquetado/scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  }
+
+  &-logo {
+    width: $tech-logo-size;
+  }
+
+  &-info {
+    @extend %equal-align;
+
+    padding: 1rem 0;
```
Comment (CarlosHWolox) [inline maquetado/scss/tech.scss]: Remember we use margin to separate from other elements and padding just to separate from borders. So, if you want more space between the element and the bottom of the card, use a margin instead.

**Pull request merged**